### PR TITLE
Camel Quarkus release 2.2.0

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: content/blog/2021/09/camel-quarkus-release-2.2.0/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the release 2.2.0 of Camel Quarkus.
+It brings Quarkus 2.2, one extension newly supported in native mode, better test coverage and documentation.
+Many thanks to all contributors and issue reporters!
+
+## Quarkus 2.2
+
+Quarkus 2.2 is focusing mainly on fix issues, improving usability and documentation.
```
Comment (aldettinger) [inline content/blog/2021/09/camel-quarkus-release-2.2.0/index.md:18]: maybe _focusing mainly on fixing issues_ ?

File: content/blog/2021/09/camel-quarkus-release-2.2.0/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+* New global flag `quarkus.devservices.enabled=false` to disable Dev Services
+* Enhanced thread model for RESTEasy Reactive
+* MongoDB service binding
+* New extension for running Narayana LRA participants
+
+Please refer to Quarkus 2.2 announcements for more details:
+* [Quarkus 2.2.1.Final](https://quarkus.io/blog/quarkus-2-2-1-final-released/)
+* [Quarkus 2.2.2.Final](https://quarkus.io/blog/quarkus-2-2-2-final-released/)
+
+## Extensions new supported in native mode
```
Comment (aldettinger) [inline content/blog/2021/09/camel-quarkus-release-2.2.0/index.md:32]: maybe _newly supported_ or _now supported_ ?

File: content/blog/2021/09/camel-quarkus-release-2.2.0/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+* [Language](/camel-quarkus/latest/reference/extensions/language.html)
+
+## Test coverage and closing functionality gaps
+
+In this release, we continued in our efforts from 2.0.0 and 2.1.0 releases to review existing tests and add new ones
```
Comment (aldettinger) [inline content/blog/2021/09/camel-quarkus-release-2.2.0/index.md:38]: maybe _we continued our efforts_ ?

File: content/blog/2021/09/camel-quarkus-release-2.2.0/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+to cover all major use case mentioned in the main Camel documentation. Check this [this GitHub issues query](https://github.com/apache/camel-quarkus/issues?q=is%3Aissue+label%3Aintegration-test+milestone%3A2.2.0+is%3Aclosed) for more details.
```
Comment (aldettinger) [inline content/blog/2021/09/camel-quarkus-release-2.2.0/index.md:39]: Check [these Github issues] ?

File: content/blog/2021/09/camel-quarkus-release-2.2.0/index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+## Documentation
+
+We have documented the [new way of managing dependencies]](/camel-quarkus/latest/user-guide/dependency-management.html#_quarkus_tooling_for_starting_a_new_project) in projects generated by [code.quarkus.io](https://code.quarkus.io/) and other [Quarkus tools](https://quarkus.io/guides/tooling).
+
+## Serialization in native mode
+
+GraalVM did not support serialitation for quite long.
```
Comment (aldettinger) [inline content/blog/2021/09/camel-quarkus-release-2.2.0/index.md:47]: serialization ?

File: content/blog/2021/09/camel-quarkus-release-2.2.0/index.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+## Breaking changes and migration steps
+
+Please refer to our [2.2.0 Migration guide](/camel-quarkus/latest/migration-guide/2.2.0.html).
+
+## Release notes
+
+Fixed issues:
+
+* [Milestone 2.2.0](https://github.com/apache/camel-quarkus/milestone/15?closed=1)
```
Comment (aldettinger) [inline content/blog/2021/09/camel-quarkus-release-2.2.0/index.md:62]: 2.2.0 must be milestone 18 ? https://github.com/apache/camel-quarkus/milestone/18?closed=1

**Review by aldettinger — APPROVED** [review]
Few typos enhancements proposed. The content is good, well done Peter :+1:

**Comment by aldettinger** [comment]
It's strange, the preview seems to be talking about @BuildTimeAvroDataFormat to be deprecated:
https://pr-619--camel.netlify.app/blog/2021/09/camel-quarkus-release-2.2.0/

Maybe there is a mix between 2.0.0 and 2.2.0 ?

**Comment by zbendhiba** [comment]
> It's strange, the preview seems to be talking about @BuildTimeAvroDataFormat to be deprecated:
> https://pr-619--camel.netlify.app/blog/2021/09/camel-quarkus-release-2.2.0/
> 
> Maybe there is a mix between 2.0.0 and 2.2.0 ?

yes there's a problem with the preview.

File: content/blog/2021/09/camel-quarkus-release-2.2.0/index.md (hunk 3 above)
Comment (ppalaga) [reply content/blog/2021/09/camel-quarkus-release-2.2.0/index.md:38]: +1, thanks

File: content/blog/2021/09/camel-quarkus-release-2.2.0/index.md (hunk 4 above)
Comment (ppalaga) [reply content/blog/2021/09/camel-quarkus-release-2.2.0/index.md:39]: +1, thanks

File: content/blog/2021/09/camel-quarkus-release-2.2.0/index.md (hunk 5 above)
Comment (ppalaga) [reply content/blog/2021/09/camel-quarkus-release-2.2.0/index.md:47]: I am really unkeyboardinated, sorry!

File: content/blog/2021/09/camel-quarkus-release-2.2.0/index.md (hunk 6 above)
Comment (ppalaga) [reply content/blog/2021/09/camel-quarkus-release-2.2.0/index.md:62]: Indeed, a leftover from the base post. Sorry.

**Comment by ppalaga** [comment]
9db599d: addressed all suggestions from @aldettinger. Thanks for the review!

**Pull request merged**